doc/code/WatchFaces.md

```diff
@@ -0,0 +1,135 @@
+# Add Watchfaces 
+
+This page will help you to add a new Watch Face to InfiniTime.
+
+## Theory
+
+This example uses the existing Digital Watch Face, copies this to a new watch face as a starting point to design a new additional watch face.
+
+## Files 
+
+- displayapp/UserApps.h
+- displayapp/apps/Apps.h.in
+- displayapp/apps/CMakeLists.txt
+- displayapp/screens/WatchFaceNew.cpp 
+- displayapp/screens/WatchFaceNew.h 
+- components/settings/Settings.h
+- CMakeLists.txt
+
+### Check out and Branch and Copy 
+
+Use git to fork and get a local copy (commands not shown). 
+
+Then with all the submodules updated add a branch and copy the watch face files
+
+```bash
+git submodule update --init
+git checkout -b Watchface_New
+cp ./src/displayapp/screens/WatchFaceDigital.cpp ./src/displayapp/screens/WatchFaceNew.cpp
+cp ./src/displayapp/screens/WatchFaceDigital.h ./src/displayapp/screens/WatchFaceNew.h
+```
+
+### Amend Files
+
+In UserApps.h add include for the new Watch Face.
+
+./src/displayapp/UserApps.h:
+
+```cpp
+
+#include "displayapp/screens/WatchFaceTerminal.h"
+#include "displayapp/screens/WatchFaceNew.h"
+```
+
+In Apps.h.in add the type from WatchFaceNew.h.
+
+./src/displayapp/apps/Apps.h.in:
+
+```cpp
+
```

Comment:
extra newline in codeblock. please remove

Reply:
removed.